DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/bridge/linux/linux_bridge_mdb_impl.py

```diff
@@ -15,6 +17,15 @@ def format_update(self, command, *argv, **kwarg):
         params = kwarg["params"]
         cmd = "bridge mdb {} ".format(command)
         ############# Implement me ################
+        mdb_entry_type = "permanent" if params.get("permanent", "") else "temp"
+        if "device" in params:
+            cmd += "dev {} ".format(params["device"])
```

Comment:
please use `dev` instead of `device`
```suggestion
        if "dev" in params:
            cmd += "dev {} ".format(params["dev"])
```